### ISSUE TITLE
Support for style overrides in articles

### ISSUE THREAD

**Issue by scottkellum** [Treesaver/treesaver#125]
Useful for ads or any page that needs styles or a completely different look and feel from the rest of the content.

**Comment by fortes** [comment]
Dupe of #51, no?

**Comment by scottkellum** [comment]
no, this is more of a feature request to support custom CSS and resources grids inside the <article> tag of any given page.

That way an advertiser would only have to supply a HTML file and image assets with out us having to modify our resources or CSS assets.

**Comment by fortes** [comment]
OK, not sure I understand completely, but re-opening.
